pub-critic/src/app/utils/spinnerStyle.css

```diff
@@ -0,0 +1,24 @@
+@keyframes spinner {
+    0% {
+      transform: rotate(0deg);
+    }
+    100% {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
This can be extracted to some common file for all CSS animations...but is up to you

Reply:
Maybe will do if I add other animations, but for now I think that this will do and is changed in later pr-s